### DISCUSSION TITLE
hybrid potential

### DISCUSSION THREAD

**Discussion by foreverfreedown** [deepmodeling/deepmd-kit#683] [Q&A]
Hi, I have a question about how we can connect two potential functions with hybrid in LAMMPS calls. I would like to ask whether the potential function trained by DeepMD can do this now. For example, I want to connect the potential function trained by DeepMD with the potential function of ZBL, how should I write my input file?
Thanks.

**Comment by tuoping** [comment]
I'd like to try the setting. Can you provide me your ZBL potential function?

**Reply by foreverfreedown** [reply]
Thank you for your reply. Lammps has its own ZBL potential function, and it only needs to set a few parameters. Is it OK to do so? Do you mean I have to build my own ZBL potential function?

**Reply by tuoping** [reply]
```bash
pair_style hybrid/overlay deepmd ../archive/graph.Mg.pb zbl 3.0 4.0 
pair_coeff 1 1  deepmd  ../archive/graph.Mg.pb
pair_coeff 1 1  zbl 12.0 12.0
```
This setting worked for me.
